### ISSUE TITLE
Scale a serivce when the image is not pulled on fresh new node

### ISSUE THREAD

**Issue by SydOps** [rancher/rancher#4133]
Rancher Version: 

    v1.0.0-rc2, maybe??
    rancher/server           latest              f2584280e144        10 days ago         768.1 MB

Docker Version: 

    Docker version 1.10.1, build 9e83765

OS: 

    rancheros 0.4.3

Steps to Reproduce: 
```
1. create two new nodes (host without any images downloaded) and add to rancher server.
2. add a service, for example, wordpress with scale count is 1, wait it actived
3. scale it to 2, the second one is never actived. 
4. manually login the second node, pull the image `wordpress`, then the wordpress container in second node running well.
```
Results:  

    need manually pull the image on fresh node, if scale the service.

Expected: 

    automatically pull image

**Comment by deniseschannon** [comment]
It looks like you are running v0.63.1 and not  v1.0.0-rc2? We haven't tagged v1.0.0-rc2 as latest in Dockerhub. 

Would you be wiling to try again and specificy v1.0.0-rc2?

**Comment by SydOps** [comment]
Seems the problem has been fixed in v1.0.0
